addons/adyen_platforms/util.py

```diff
@@ -40,3 +56,44 @@ def __call__(self, request):
         })
 
         return request
+
+def odoo_payments_proxy_control():
+    def decorator(func):
```

Comment:
Is it useful to make this a two-level decorator? Are there chances it will need to take arguments in the future?

Also can use `decorator.decorator` to simplify this a bit.